### ISSUE TITLE
Cannot add ticket when ticket has a location field on the layout

### ISSUE THREAD

**Issue by githubbob42** [githubbob42/mingle2github2#1493]
**[Mingle Card: 1701](https://liquidframeworks.mingle.thoughtworks.com/projects/alpine_mobile/cards/1701)**
Steps to reproduce
------------------

```
| | |
|-|-|
|**Version #**|
|1.0.1.1854|
|**OS**|
| |
|**Browser**|
| |
|**Username**|
|bobfield@mobileteam.com |
|**Password**|
|computer#4|

  

 


```

1.  Add a location field to the ticket layout in console
2.  Sync Mobile 
3.  Try to add a ticket 
4.  Sync

Expected result
---------------

_You should see the new ticket appear in console once the sync is complete._

Actual Result
-------------

_The sync fails when processing the sync packet. A second sync will show that the new ticket was never created._

Related Cards
-------------

Test Plan
---------

**Comment by githubbob42** [comment]
### Related Story: #1394 Field User records picking up wastewater at unmanned site ([Mingle](https://liquidframeworks.mingle.thoughtworks.com/projects/alpine_mobile/cards/1587))
